debian/vala-panel/debian/control

```diff
@@ -2,126 +2,74 @@ Source: vala-panel
 Section: x11
 Priority: optional
 Maintainer: Konstantin P. <ria.freelander@gmail.com>
-Build-Depends: debhelper (>= 9), cmake (>= 2.8.0), valac (>= 0.24.0), libpeas-dev (>= 1.2.0), libgtk-3-dev (>= 3.12.0), libwnck-3-dev (>= 3.4.0), docbook-to-man
-Standards-Version: 3.9.6
+Build-Depends: cmake (>= 2.8.0),
+               debhelper (>= 9),
+               docbook-to-man,
+               libgtk-3-dev (>= 3.12.0),
```

Comment:
Here must be >= 3.22.0 (or 3.21.X), because it uses 3.22 APIs (gtk_menu_popup_for_widget, GdkMonitor and so... )